pkg/controllers/management/cluster/clustercontroller.go

```diff
@@ -33,7 +33,7 @@ const (
 	AzureL4LB               = "Azure L4 LB"
 	NginxIngressProvider    = "Nginx"
 	DefaultNodePortRange    = "30000-32767"
-	capabilitiesAnnotation  = "capabilities.cattle.io/"
+	capabilitiesAnnotation  = "capabilities/"
```

Comment:
Was this change unintentional?

Reply:
yes, we want the user to be able to change this via rancher but currently they cannot because norman filters out annotations with cattle.io in them.

Reply:
I couldn't figure out the context based on the info provided :) ... You were fixing this as well I guess: https://github.com/rancher/rancher/issues/21039

Reply:
That is correct. The intention of that was to provide a way for users to override capabilities by using annotations. We decided it should be bundled in with this PR as it operates on the same area and requires that logic.